src/formats/palette.h

```diff
@@ -22,6 +22,28 @@
 extern "C" {
 #endif
 
+// All sprites have holes, these holes need to be transparent.
+#define SPRITE_TRANSPARENT_INDEX 0
+
+// Backgrounds have no transparent pixels and need all pixels to be rendered.
+#define BACKGROUND_TRANSPARENT_INDEX -1
+
+// Force an opaque menu for now.
+#define MENU_TRANSPARENT_INDEX -1
+
+// Portraits have a transparancy that needs to offset past other items in the global palette.
+#define PORTRAIT_TRANSPARENT_INDEX 0xD0
+
+// Portrait highlights need a special offset for transparency to not conflict with other entries in the global palette.
+#define HIGHLIGHT_TRANSPARENT_INDEX 90
```

Comment:
Actually, why does HIGHLIGHT and PORTRAIT need other transparency from 0 ?

Reply:
It breaks for me with different transparency indices. Something to look into later.